bin/cylc-help

```diff
@@ -422,6 +424,8 @@ comsum['broadcast'] = 'Change suite [runtime] settings on the fly'
 comsum['jobs-kill'] = '(Internal) Kill task jobs'
 comsum['jobs-poll'] = '(Internal) Retrieve status for task jobs'
 comsum['jobs-submit'] = '(Internal) Submit task jobs'
+comsum['remote-init'] = '(Internal) Initialise a task remote'
+comsum['remote-tidy'] = '(Internal) Tidy a task remote'
```

Comment:
We should document this terminology, which I like, and which I guess comes from task `[[[remote]]]`, and use it consistently - perhaps in another PR though. 

I've mostly been using the uglier "task job host account" or similar, I think.  Something like this?: *a "task remote" is a user account, other than the suite host account, where a task job is submitted to run. It can be on the suite host machine or another machine.*

Reply:
Can we do this in a follow-on PR? (This PR has lasted long enough for more changes.)

(The `[user@]host` syntax is sometimes called an *authority* - see https://en.wikipedia.org/wiki/URL for example - but everyone are confused by the phrase.)

Reply:
Yes, that's fine - and sorry for the review delay on this!